src/store/items/actions.js

```diff
@@ -173,8 +173,23 @@ export default {
         commitFromRoot('updateItemByTemporaryId', responseItem);
         dispatchFromRoot('cache/_saveItemInCache', responseItem);
 
+        // eslint-disable-next-line no-param-reassign
+        delete responseItem.temporaryId;
+        
         if (getters.currentItemObj) {
-          commit('setCurrentItemObj', responseItem);
+          const fieldsToUpdate = [
+            'createdAt',
+            'updatedAt',
+            'listId',
+            'userId',
+            'deletedAt',
+            'id',
+            'temporaryId',
+          ];
+
+          fieldsToUpdate.forEach(field => {
+            commit('updateItemFieldLocally', { field, value: responseItem[field] });
```

Comment:
I think it's better to make some another mutation which can take or one or many fields with values